frontend/src/components/Card.tsx

```diff
@@ -119,56 +138,30 @@ const Card = ({
             />
           ))}
         </div>
-        {!social || social.length === 0 ? (
-          <div
-            className={cn(
-              'mt-3 flex items-center pr-4',
-              isMobile && 'mt-4 w-full justify-end pr-4'
-            )}
-          >
-            <ActionButton tooltipLabel={tooltipLabel} url={button.url} onClick={button.onclick}>
-              {button.icon}
-              {button.label}
-            </ActionButton>
-          </div>
-        ) : (
-          <div
-            className={cn(
-              'flex w-full flex-wrap items-center justify-between gap-6',
-              isMobile && 'items-start'
-            )}
-          >
-            <div
-              className={cn('flex w-full items-center justify-between', isMobile && 'flex-wrap')}
-            >
-              {/* Render social links if available */}
-              {social && social.length > 0 && (
-                <HStack id="social" mt={2}>
-                  {social.map((item) => (
-                    <Link
-                      key={`${item.title}-${item.url}`}
-                      href={item.url}
-                      target="_blank"
-                      rel="noopener noreferrer"
-                      display="flex"
-                      alignItems="center"
-                      gap={2}
-                    >
-                      <FontAwesomeIcon icon={getSocialIcon(item.url)} className="h-5 w-5" />
-                    </Link>
-                  ))}
-                </HStack>
-              )}
-              {/* Action Button */}
-              <div className="flex items-center">
-                <ActionButton tooltipLabel={tooltipLabel} url={button.url} onClick={button.onclick}>
-                  {button.icon}
-                  {button.label}
-                </ActionButton>
-              </div>
-            </div>
-          </div>
-        )}
+
+        <div className="flex w-full items-center justify-between">
+          {social && social.length > 0 && (
+            <HStack id="social" mt={2}>
+              {social.map((item) => (
+                <Link
+                  key={`${item.title}-${item.url}`}
+                  href={item.url}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  display="flex"
+                  alignItems="center"
+                  gap={2}
+                >
+                  <FontAwesomeIcon icon={getSocialIcon(item.url)} className="h-5 w-5" />
+                </Link>
+              ))}
+            </HStack>
+          )}
+          <ActionButton tooltipLabel={tooltipLabel} url={button.url} onClick={button.onclick}>
```

Comment:
Can we keep the button on the right as it was before? 🤔 
![Screenshot 2025-03-30 at 7 13 54 PM](https://github.com/user-attachments/assets/86effddf-8bd8-4900-a5bc-597b2bab2724)
![Screenshot 2025-03-30 at 7 20 07 PM](https://github.com/user-attachments/assets/4d72d549-c673-4c5c-8392-5a7a1b6bb0f8)

They should be on the same level as labels, kinda dividing that div into 2 parts for labels/topics and button.